cs-minibot-platform-src/src/main/resources/public/js/view.js

```diff
@@ -355,4 +421,91 @@ window.addEventListener("keydown", function(e) {
     }
 }, false);
 
+function displayOccupancyMatrix(height, width, size) {
+
+    $.ajax({
+        method: "POST",
+        url: '/getOccupancyMatrix',
```

Comment:
it's a little odd that we are doing a POST to GET something- probably should rename one or the other